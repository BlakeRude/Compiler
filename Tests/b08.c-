// C-S21
main()
{
     outputb(not true);
     outnl();
     outputb(not false);
     outnl();
}
