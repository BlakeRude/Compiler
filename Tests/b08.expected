Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: b08.tm
F
T
Bye.
